privaterelay/settings.py

```diff
@@ -48,7 +48,7 @@
 # defaulting to blank to be production-broken by default
 SECRET_KEY = config("SECRET_KEY", None, cast=str)
 
-ON_HEROKU = config("ON_HEROKU", False, cast=bool)
+RELAY_CHANNEL = config("RELAY_CHANNEL", "prod", cast=str)
```

Comment:
Rather than having a separate env variable for the different environment, we should use the `settings.SITE_ORIGIN` to set the `RELAY_CHANNEL`, this will remove the duplicate check on whether or not the environment is prod, stage, dev, or local.
```suggestion
ORIGIN_ENV_MAP = {"dev.fxprivaterelay.nonprod.cloudops.mozgcp.net": "dev", "stage.fxprivaterelay.nonprod.cloudops.mozgcp.net": "stage", "relay.firefox.com": "prod"}
RELAY_CHANNEL = ORIGIN_ENV_MAP[SITE_ORIGIN] if SITE_ORIGIN in ORIGIN_ENV_MAP else "local"
```

Reply:
According to the infra document seems like the `https://` needs to be added: https://github.com/mozilla-services/cloudops-infra/blob/b9e9796555e0e05aaf4a1a2e5e60c8cbbdbe0ff8/projects/fxprivaterelay/k8s/values/app.yaml#L45

Reply:
Oh nice! Happy to not add another env var. I'll update.

Reply:
Updated.

Reply:
Something like:

```python
_DEFAULT_FOR_CHANNEL = {
    "prod": {
        "SITE_ORIGIN": "https://relay.firefox.com",
    },
    "stage": {
        "SITE_ORIGIN": "https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net",
    },
    "dev": {
        "SITE_ORIGIN": "https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net",
    },
    "local": {
        "SITE_ORIGIN": "http://127.0.0.1:8000",
    }
}

def config_with_channel_default(name, default=None, cast = None)
    if RELAY_CHANNEL in _DEFAULT_FOR_CHANNEL:
        default = _DEFAULT_FOR_CHANNEL[RELAY_CHANNEL][name]
    if cast is not None:
        return config(name, default, cast=cast)
    else:
        return config(name, default)

SITE_ORIGIN = config_with_channel_default("SITE_ORIGIN", None)
```

Reply:
Can we pull `RELAY_CHANNEL` from `SITE_ORIGIN` For Now™ so we can merge + deploy this without needing SRE to do anything?